tests/test_client.py

```diff
@@ -115,7 +124,6 @@ def test_coro():
         self.loop.run_until_complete(test_coro())
         if future.exception():
             raise future.exception()
-            raise future.exception()
```

Comment:
This is not strictly related to the sense of the MR. It just struck me and slipped into this MR.